src/model/UserphraseModel.cpp

```diff
@@ -106,14 +107,15 @@ void UserphraseModel::remove(QModelIndexList indexList)
             userphrase_[index].bopomofo_.toUtf8().constData());
         if (ret > 0) {
             removerecord_.push_back(Userphrase{userphrase_[index]});
-            // FIXME: std::vector::erase is an inefficient operation.
+            remove_cnt++;
+			// FIXME: std::vector::erase is an inefficient operation.
             userphrase_.erase(userphrase_.begin() + index);
         } else {
             qWarning() << "chewing_userphrase_remove() returns" << ret;
         }
         // FIXME: Handle chewing_userphrase_remove fails.
     }
-
+	maxundocnt = remove_cnt;
```

Comment:
Did you mean I add a new STL like vector and don't use a simple global variable?